# weight_value

weight_value :
  1 : 1.0		# r_arm_sh_p1
  2 : 1.0		# l_arm_sh_p1
  3 : 1.0		# r_arm_sh_r
  4 : 1.0		# l_arm_sh_r
  5 : 1.0		# r_arm_sh_p2
  6 : 1.0		# l_arm_sh_p2
  7 : 3.0		# r_arm_el_y
  8 : 3.0		# l_arm_el_y
  9 : 1.0		# r_arm_wr_r
  10 : 1.0		# l_arm_wr_r
  11 : 10.0		# r_arm_wr_y
  12 : 10.0		# l_arm_wr_y
  13 : 10.0		# r_arm_wr_p
  14 : 10.0		# l_arm_wr_p
  15 : 1.0		# r_leg_hip_y
  16 : 1.0		# l_leg_hip_y
  17 : 1.0		# r_leg_hip_r
  18 : 1.0		# l_leg_hip_r
  19 : 1.0		# r_leg_hip_p
  20 : 1.0		# l_leg_hip_p
  21 : 1.0		# r_leg_kn_p
  22 : 1.0		# l_leg_kn_p
  23 : 1.0		# r_leg_an_p
  24 : 1.0		# l_leg_an_p
  25 : 1.0		# r_leg_an_r
  26 : 1.0		# l_leg_an_r
  27 : 5.0		# torso_y
  28 : 1.0		# head_y
  29 : 1.0		# head_p
  30 : 1.0		# l_arm_grip
  31 : 1.0		# r_arm_grip

